File: src/main/java/no/nav/domain/pensjon/regler/gomregning/PersongrunnlagOmregningGrunnlag.java
package no.nav.domain.pensjon.regler.gomregning;

import no.nav.domain.pensjon.regler.grunnlag.Inntektsgrunnlag;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class PersongrunnlagOmregningGrunnlag implements Serializable {


    private Long persongrunnlagId;
    private List<Inntektsgrunnlag> inntektsgrunnlagGrunnlagListe;
    private boolean lonnsvekstOmregn;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/gomregning/PersongrunnlagOmregningGrunnlag.kt
package no.nav.pensjon.regler.domain.gomregning

import no.nav.pensjon.regler.domain.grunnlag.Inntektsgrunnlag
import java.io.Serializable

class PersongrunnlagOmregningGrunnlag : Serializable {
    var persongrunnlagId: Long? = null
    var inntektsgrunnlagGrunnlagListe: MutableList<Inntektsgrunnlag> = mutableListOf()
    var lonnsvekstOmregn = false
}